src/sbd-md.c

```diff
@@ -108,6 +109,7 @@ open_device(const char* devname, int loglevel)
 
 	if (io_setup(1, &st->ioctx) != 0) {
 		cl_perror("io_setup failed");
+		free(st->buffer);
```

Comment:
This isn't necessary since st is allocated just above here, and buf hasn't been set yet

Reply:
But doesn't hurt either ;-)
Added when I was still undecided what to do first.
Anyway whole thing wasn't meant to be reviewed yet - just wanted to see posix_memalign being handled by all the targets CI builds against.